internal/runtimeconfig/runtime_config.go

```diff
@@ -28,7 +28,7 @@
 //	    type: "redis"
 //	    options:
 //	      - name: url
-//	        value: "redis://localghost:9000"
+//	        value: "redis://localhost:9000"
```

Comment:
I'm not certain about this. It's possible this is being cute and refers to a computer that is _not_ `localhost`. But I didn't see anything in this repository that indicated it liked being "cute"...

Reply:
This _was_ deliberate 😅  (partially to indicate that this isn't "real" config)

Reply:
Might I suggest using `something` instead?

Reply:
i'd rather keep it fun, but don't really care (the rest of the changes lgtm fwiw)